extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -826,9 +825,9 @@ export class AjaxMonitor extends BaseTelemetryPlugin implements IDependenciesPlu
                 try {
                     const responseHeadersString = xhr.getAllResponseHeaders();
                     if (responseHeadersString !== null) {
-                        const index = _indexOf(responseHeadersString.toLowerCase(), RequestHeaders.requestContextHeaderLowerCase);
+                        const index = _indexOf(responseHeadersString.toLowerCase(), RequestHeaders[eRequestHeaders.requestContextHeaderLowerCase]);
```

Comment:
So this `RequestHeaders[eRequestHeaders.requestContextHeaderLowerCase]`

becomes `RequestHeaders[8 /* requestContextHeaderLowerCase */]`

which becomes `J[8]`
Previously this would have become `J.requestContextHeaderLowerCase`